ivy/functional/backends/paddle/statistical.py

```diff
@@ -8,7 +8,6 @@
 import ivy
 from ivy.func_wrapper import (
     with_supported_dtypes,
-    with_unsupported_dtypes,
```

Comment:
I may have mistakenly deleted this line but  there is no function in this file which is using `@with_unsupported_dtypes` decorator, so ig its fine.

Reply:
yep sounds good!